tests/api/test_record_permissions.py

```diff
@@ -73,45 +74,6 @@ def login_and_test(user_id):
     login_and_test(users["admin"].id)
 
 
-@pytest.mark.parametrize(
```

Comment:
I think we should leave those to check the explicit permissions anyway. it helps if we change something in invenio-access

Reply:
I removed them because to me they look exactly the same as the previous one, so a duplication. Can you confirm?